client/src/components/NavLink/NavLink.stories.js

```diff
@@ -1,24 +1,28 @@
-import React from 'react';
-import { storiesOf } from '@storybook/react';
-import NavLink from './NavLink';
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import NavLink from "./NavLink";
 
 const placeholderTextProp = "community";
 const placeholderUrl = "/";
 const placeholderDropdownProp = [
   {
-    text: 'Explore People',
-    to: '/'
+    text: "Explore People",
+    to: "/"
   },
   {
-    text: 'Join Groups',
-    to: '/'
+    text: "Join Groups",
+    to: "/"
   }
-]
+];
 
-storiesOf('NavLink', module)
+export default {
+  title: "Layout/NavLink"
+};
+
+storiesOf("Layout/NavLink", module)
```

Comment:
I think NavLink and NavLinkDropdownMenu should maybe go under Atoms and if possible group them together somehow under Atoms